### DISCUSSION TITLE
How to disable the keymapping for the ACE editor?

### DISCUSSION THREAD

**Discussion by thewindgods** [brookhong/Surfingkeys#2208] [Q&A]
The ACE editor is not something that I typically use, and it interferes with my typing by overriding the common Ctrl-i making text _italic_.

Unmapping it or rebinding it in my Surfingkeys settings has not worked.

**Comment by pablospe** [comment]
I tried with this but didn't work:
`api.unmap('<Ctrl-i>');`
